tpgtools/resource.go

```diff
@@ -392,24 +392,12 @@ func createResource(schema *openapi.Schema, typeFetcher *TypeFetcher, overrides
 		versionMetadata:      version,
 		Description:          schema.Description,
 		location:             location,
-		InsertTimeoutMinutes: 10,
-		UpdateTimeoutMinutes: 10,
-		DeleteTimeoutMinutes: 10,
+		InsertTimeoutMinutes: 30,
```

Comment:
We'll want to bump the #s in the documentation as well!

Reply:
Whoops, yeah, this already made the right change! I thought this was resouce.go.tmpl and not resource.go.

Reply:
Awesome. Thanks!